src/main/scala/io/codeheroes/kafkalogger/ServiceDetails.scala

```diff
@@ -0,0 +1,8 @@
+package io.codeheroes.kafkalogger
+
+final case class ServiceDetails(hostname: String,
+                                host: String,
+                                service: String,
+                                version: String,
+                                task: String)
```

Comment:
Task should be optional IMO and in parsing to json can be converted to "" if empty.